spec/features/reviews/new_spec.rb

```diff
@@ -30,4 +32,28 @@
     expect(page).to have_content('Title: Good shelter')
     expect(page).to have_content('Content: It was a great shelter')
   end
+
+  it "does not save new review without all the information" do
+
+    visit "/shelters/#{@shelter_2.id}/reviews/new"
+
+    fill_in :title, with: 'Good shelter'
+    fill_in :rating, with: '5'
+
+    click_button 'Submit'
+
+    expect(current_path).to eq("/shelters/#{@shelter_2.id}/reviews/new")
```

Comment:
checks that an error message is displayed when only some of the fields are entered.